src/util/saveArticles.ts

```diff
@@ -915,6 +927,13 @@ function isSubpage(id: string, mw: MediaWiki) {
     return false;
 }
 
+function addWebpScript() {
+    return `<script>
+	        var webpMachine = new webpHero.WebpMachine()
+	        webpMachine.polyfillDocument()
+        </script>`
```

Comment:
Make a one line DOM not and indent it properly so this is aligned with the rest. `<script>var webpMachine = new webpHero.WebpMachine(); webpMachine.polyfillDocument();</script>`